src/index.ts

```diff
@@ -21,20 +21,22 @@ import {
   elementGetAnimations,
   documentGetAnimations,
   ProxyAnimation
-} from "./proxy-animation.js";
+} from "./proxy-animation";
 
 import { initCSSPolyfill } from "./scroll-timeline-css"
 
-function initPolyfill() {
+export { ScrollTimeline, ViewTimeline }
+
+function initPolyfill(): void {
   // initCSSPolyfill returns true iff the host browser supports SDA
   if (initCSSPolyfill()) {
     return;
   }
 
-  if ([...document.styleSheets].filter((s) => s.href !== null).length) {
+  if ([ ...document.styleSheets ].filter((s) => s.href !== null).length) {
```

Comment:
I may of had the `pretty: true` flag turned on momentarily that led to this autoformatting, happy to remove it.